src/components/strategies/create/overlapping/CreateOverlappingStrategy.tsx

```diff
@@ -44,6 +44,15 @@ export type SetOverlappingParams = (
   max: string
 ) => Promise<StrategyPrices | undefined>;
 
+const getInitialPrice = (marketPrice: SafeDecimal, quote: Token) => {
```

Comment:
is this only for quote? Might be worth to rename `quote` to `tkn` and make it a bit more universal.

Reply:
All price are display in quote in the app so it'll always be quote